src/pages/CreateCirclePage/CreateCirclePage.tsx

```diff
@@ -41,6 +40,7 @@ export const SummonCirclePage = () => {
     avatar: string;
     avatarRaw: File | null;
   }>({
+    // i don't know who to remove this getCircleAvatar in line 44
```

Comment:
i think this can just be `null`. if you look at where `logoData.avatar` is used, it's used in `<Avatar>` on line 194, so if you just set the `name` property on that, it should work fine